### PR TITLE
HBW-220 Add spinner on tasks loading

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
Nov-27-2019 11-25-01
Nov-27-2019 11-38-37

**Comment by mxsmirnoff** [comment]
Nov-27-2019 11-38-37

File: hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ const withTaskListContext = (WrappedComponent) => {
       claimingTask: null,
       page:         1,
       lastPage:     false,
-      fetched:      false,
-      fetching:     false,
+      fetching:     true,
+      searching:    false,
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: 👍

File: hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx [hunk 2]
```diff
@@ -140,6 +141,7 @@ const withTaskListContext = (WrappedComponent) => {
         ...this.state,
         tabs:              this.tabs,
         fetching:          this.state.fetching,
+        searching:         this.state.searching,
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: look at line 141 :)

File: hbw/app/javascript/packs/hbw/components/claiming/short_list.jsx [hunk 3]
```diff
@@ -37,6 +39,10 @@ const HBWClaimingShortList = ({ env, tasks, fetched }) => {
       </div>
   );
 
+  const renderLoader = () => (
+    <div className='loader'><i className="fas fa-spinner fa-spin fa-2x"></i></div>
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/short_list.jsx]: We have <Pending /> component, right?

File: hbw/app/javascript/packs/hbw/components/claiming/table.jsx [hunk 4]
```diff
@@ -194,7 +200,8 @@ class HBWTasksTable extends Component {
             {tasks.map((row, index) => this.renderRow(row, index))}
           </tbody>
         </table>
-        {fetched && tasks.length === 0 && this.renderNoTasks()}
+        {!fetching && tasks.length === 0 && this.renderNoTasks()}
+        {fetching && this.renderLoader()}
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/table.jsx]: Let's swap it for logic sake)

**Pull request merged**